packages/app-dashboard/src/components/withdraw/WalletConnect/WalletConnectCard.tsx

```diff
@@ -0,0 +1,64 @@
+import { ReactNode } from 'react';
+
+interface WalletConnectCardProps {
+  children: ReactNode;
+  variant: 'sessions' | 'requests' | 'proposal';
+  title: string;
+  icon?: string;
+  subtitle?: string;
+  className?: string;
+}
+
+export function WalletConnectCard({
+  children,
+  variant,
+  title,
+  icon,
+  subtitle,
+  className = '',
+}: WalletConnectCardProps) {
+  const variantStyles = {
+    sessions: {
+      gradient: 'bg-gradient-to-r from-blue-50 to-indigo-50',
+      border: 'border-blue-100',
+      text: 'text-blue-700',
+      titleText: 'text-blue-900',
+      titleBorder: 'border-blue-100',
+    },
+    requests: {
+      gradient: 'bg-gradient-to-r from-orange-50 to-amber-50',
+      border: 'border-orange-100',
+      text: 'text-orange-800',
+      titleText: 'text-orange-900',
+      titleBorder: 'border-orange-100',
+    },
+    proposal: {
+      gradient: 'bg-gradient-to-r from-yellow-50 to-amber-50',
+      border: 'border-yellow-100',
+      text: 'text-yellow-800',
+      titleText: 'text-yellow-900',
+      titleBorder: 'border-yellow-100',
+    },
+  };
```

Comment:
Remove this out of the component function. Having it here will redefine the object on every render and make every child render that uses it re-render even when their props don't change